example/ubuntu-22.04-prepare.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/zsh
+
+set -eu
+
+if [ ! -f ubuntu-22.04.1-live-server-arm64.iso ]; then
+	echo "Downloading image..."
+	curl -fSLO https://cdimage.ubuntu.com/releases/22.04/release/ubuntu-22.04.1-live-server-arm64.iso
+fi
+
+if [ -d cidata ]; then
+	rm -rf cidata*
+fi
+
+mkdir cidata
+touch cidata/meta-data
+cat << 'EOF' > cidata/user-data
+#cloud-config
+autoinstall:
+  version: 1
+  identity:
+    hostname: ubuntu-server
+    password: "$6$exDY1mhS4KUYCE/2$zmn9ToZwTKLhCw.b4/b.ZRTIZM30JZ4QrOQ2aOXJ8yk96xpcCof0kxKwuX1kqLG/ygbJ1f8wxED22bTL4F46P0"
```

Comment:
What's this password? Should it be `ubuntu`?

Reply:
> Should it be `ubuntu`?

Well it is:
```
echo 'ubuntu' |openssl passwd -6 -stdin -salt exDY1mhS4KUYCE/2
$6$exDY1mhS4KUYCE/2$zmn9ToZwTKLhCw.b4/b.ZRTIZM30JZ4QrOQ2aOXJ8yk96xpcCof0kxKwuX1kqLG/ygbJ1f8wxED22bTL4F46P0
```